src/main/java/fruitshop/Main.java

```diff
@@ -0,0 +1,49 @@
+package fruitshop;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.service.DataConverter;
+import fruitshop.service.FileReader;
+import fruitshop.service.FileWriter;
+import fruitshop.service.ReportGenerator;
+import fruitshop.service.ShopService;
+import fruitshop.service.impl.DataConverterImpl;
+import fruitshop.service.impl.FileReaderImpl;
+import fruitshop.service.impl.FileWriterImpl;
+import fruitshop.service.impl.ReportGeneratorImpl;
+import fruitshop.service.impl.ShopServiceImpl;
+import fruitshop.strategy.BalanceOperation;
+import fruitshop.strategy.OperationHandler;
+import fruitshop.strategy.OperationStrategy;
+import fruitshop.strategy.OperationStrategyImpl;
+import fruitshop.strategy.PurchaseOperation;
+import fruitshop.strategy.ReturnOperation;
+import fruitshop.strategy.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
+        FileReader fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("src/main/resources/reportToRead.csv");
```

Comment:
Avoid using hardcoded file paths. It's better to pass file paths as parameters or constants, so the code is more flexible and testable.